src/KeyVault/KeyVault/Commands/Certificate/ImportAzureKeyVaultCertificate.cs

```diff
@@ -153,16 +221,32 @@ public override void ExecuteCmdlet()
                 ValidateParameters();
 
                 PSKeyVaultCertificate certBundle = null;
+                PSKeyVaultCertificatePolicy policy = null;
+                if (!string.IsNullOrEmpty(PolicyPath))
+                {
+                    policy = PSKeyVaultCertificatePolicy.FromJsonFile(PolicyPath);
+                }
+                else if ( PolicyObject != null)
+                {
+                    policy = PolicyObject;
+                }
+                
+                if ( policy != null && policy.SecretContentType != ContentType)
```

Comment:
1. The default value of ContentType is Constants.Pkcs12ContentType. If customer uses a PEM file as input, ContentType is Constants.Pkcs12ContentType in this case but we should not throw exception.
2. Please move validation to  ValidateParameters();
```suggestion
                if ( policy != null && this.IsParameterBound(c => c.ContentType) && policy.SecretContentType != ContentType)
```

Reply:
1. Should we remove the default value of ConetentType?
2. Moved.